sample/src/main/java/org/aerogear/graphqlandroid/GenericMutation.kt

```diff
@@ -0,0 +1,229 @@
+package org.aerogear.graphqlandroid
+
+import android.util.Log
+import com.apollographql.apollo.api.*
+import com.apollographql.apollo.api.internal.UnmodifiableMapBuilder
+import com.apollographql.apollo.api.internal.Utils
+import io.reactivex.internal.util.BackpressureHelper.add
+import org.aerogear.graphqlandroid.GenericMutation.Data.Mapper
+import org.aerogear.graphqlandroid.GenericMutation.Variables
+import org.aerogear.graphqlandroid.type.CustomType
+import java.lang.String.format
+import java.lang.StringBuilder
+import java.util.*
+import kotlin.collections.ArrayList
+
+class GenericMutation(
```

Comment:
Do we still need these files for this PR in particular?

Reply:
@harshithdwivedi Please have a look.